ingress/elb/elb.go

```diff
@@ -181,6 +190,13 @@ func (e *elb) Detach() error {
 	return nil
 }
 
+func (e *elb) Health() error {
+	if e.registeredFrontends != e.expectedFrontends {
```

Comment:
And to provide visibility to the write. I will add a lock around Attach and Health